src/Common/ExceptionHandler/ExceptionHandler_posix.cpp

```diff
@@ -1,9 +1,43 @@
 #include <signal.h>
 #include <execinfo.h>
 #include <string.h>
+#include <string>
 
 #include "config/CemuConfig.h"
 
+void demangleAndPrintBacktrace(char** backtrace, size_t size)
+{
+	for (char** i = backtrace; i < backtrace + size; i++)
+	{
+		std::string traceLine{*i};
+		size_t parenthesesOpen = traceLine.find_last_of('(');
+		size_t parenthesesClose = traceLine.find_last_of(')');
+		size_t offsetPlus = traceLine.find_last_of('+');
+		if (!parenthesesOpen || !parenthesesClose || !offsetPlus ||
+			 offsetPlus < parenthesesOpen || offsetPlus > parenthesesClose)
+		{
+			// something unexpected was read. fall back to default string
+			std::cerr << traceLine << std::endl;
+			continue;
+		}
+
+		std::string symbolName = traceLine.substr(parenthesesOpen+1,offsetPlus-parenthesesOpen-1);
+		int status = -1;
+		char* demangled = abi::__cxa_demangle(symbolName.c_str(), nullptr, nullptr, &status);
```

Comment:
This is GNU libstdc++ specific, shouldn't it be `ifdef`d? I believe macOS users use LLVM libc++

Reply:
Clang implements the same function so it should be okay. GitHub CI does build for MacOS and they seem to have passed so I don't expect any issues.

Reply:
Does this still work when compiling with `-stdlib=libc++`?

Reply:
I tried to compile with that option but a vcpkg library is compiled without it causing linker errors. I don't know how to change the compiler options for vcpkg dependencies.

Reply:
I found out how to change the compiler option for vcpkg libraries but I can't use clang to compile them and my GCC installation is not compiled with -stdlib support. So I can't test this. However a bigger problem for this code on Mac OS is problably the format of the output of backtrace_symbols. Best case scenario if it's different the sanity checks will probably fail and it will just print the raw output.